sudo/src/main.rs

```diff
@@ -88,6 +31,45 @@ do this then this software is not suited for you at this time."
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[derive(Default)]
+pub struct SudoersPolicy;
```

Comment:
I don't like the fact that this is a unit struct (clippy is also complaining about it in the GitHub review); this creates several structs/traits that have "Policy" in the name (PolicyPlugin, SudoersPolicy, Sudoers, Judgement, PreJudgementPolicy). I think it's better to either just implement this for sudoers::Sudoers directly, since it is essentially a facade for its own interface.

And in fact, in the future -- the PolicyPlugin trait should probably be moved to a spot where the sudoers crate can see it and implement that trait directly for the sudoers object (we shouldn't get too much in the habit of building abstractions for other abstractions we also wrote ourselves).